### DISCUSSION TITLE
HOW TO SET THE IMAGE THAT DISPLAYS WHEN YOUR WEBSITE IS SHARED ON SOCIAL MEDIA

### DISCUSSION THREAD

**Discussion by dinakar17** [timlrx/tailwind-nextjs-starter-blog#529] [Ideas]
I'm wondering how to control the make feature image different for every blog post that we can share on social media. Any suggestions/ ideas regarding this?
![image](https://user-images.githubusercontent.com/85105044/185781551-10fe663e-fe53-498b-9c4c-75e559c25926.png)

**Comment by timlrx** [comment]
As mentioned in the [README](https://github.com/timlrx/tailwind-nextjs-starter-blog#post), you can configure the `images` frontmatter field for every blog post to customize the social media display pic
>images (optional, if none provided defaults to socialBanner in siteMetadata config)

**Reply by dinakar17** [reply]
Sorry! Didn't see that! :( Thanks for your time
